### ISSUE TITLE
Include the simplest possible example in README

### ISSUE THREAD

**Issue by plantroon** [nilsmeyer/ansible-debootstrap#28]
Could you include a simple example on how to make a simple directory debootstrap with this script? That would make it more complete and easier to try out.

I may do a PR when (or if) I figure it out. Is it a supported scenario to do an installation to a directory? It is especially useful for LXC containers.

**Comment by nilsmeyer** [comment]
It's not a supported scenario - I think the approach for just bootstrapping to a directory for use as lxc container could be made quite simple, a lot of the code in this repo is about setting up a bare-metal machine (or VM).

**Comment by plantroon** [comment]
> It's not a supported scenario - I think the approach for just bootstrapping to a directory for use as lxc container could be made quite simple, a lot of the code in this repo is about setting up a bare-metal machine (or VM).

Yes, it can be done easier, you're right. I had different expectations when I asked back then lol.
